data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+	minor
+	name `Bad Harvest`
+	description `The pirate world of <destination> is experiencing a famine. Warlords of the planet have threatened the Syndicate to give them food, and the Syndicate has agreed. Deliver <cargo> to the world for <payment>.`
+	source
+		near "Almach" 2 4
+	destination "Deadman's Cove"
+	cargo "food" 287
+	to offer
+		random < 20
+		"combat rating" > 400
+	
+	on offer
+		conversation
+			`In the <origin> spaceport, you are approached by a Syndicate employee. By the looks of his uniform, he works for Syndicated Security. "Hello, Captain. You're a compitent pilot, correct?" You nod your head. "Excellent. The Syndicate needs <cargo> transported to <destination>. The pirate world has seen a particularly bad harvest this past season, so much of the population is without sufficient food. Pirate warlords have threatened to begin raiding our freighters and frontier worlds if we don't provide them with food. It's cheaper to do what they want, so we've agreed to the deal, but we don't want to risk any of our own ships being destroyed. The Syndicate will pay you <payment> should you accept and complete this job."`
+			choice
+				`	"Alright, I'll get my ship ready."`
+					accept
+				`	"Sorry, but I have better things to be doing."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Large Core Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Schedar"
+		fleet "Large Core Pirates"
+		fleet "Small Core Pirates" 3
+	
+	on complete
+		payment
+		payment 10000
+		dialog `Multiple bands of pirate each run by a different warlord are waiting in the spaceport when you land. They take the food from your cargo hold like animals, leaving quite the mess behind. When you return to your ship, you find that the Syndicate has deposited <payment> into your account.`
+
+
+
+mission "Pirate Blockade"
+	minor
+	name `Escape to Freedom`
+	description `Bring <bunks> escaped slaves to freedom from their pirate warlords on <destination>. The woman on <origin> promised that someone would give you <payment> for doing this.`
+	source "Buccaneer Bay"
+	destination "Hippocrates"
+	passengers 19
+	to offer
+		"combat rating" > 500
+	
+	on offer
+		conversation
+			`It is dusk at the <origin> spaceport, and the setting sun casts long shadows across the spaceport deck. You take notice of a slim cloaked figure walking only a few feet behind you who seems to be keeping pace with you.`
+			choice
+				`	(Confront them.)`
+					goto confront
+				`	(Stop walking and see if they stop as well.)`
+			
+			`	You stop be the entrance to a store that claims to have "the best Polecats!" and lean against the building, trying not to draw attention to yourself. The figure keeps walking at the same pace, and as they pass you a soft voice says "Follow me," from under the cloak before the figure bolts off into a dark alleyway next to the building.`
```

Comment:
There should be a comma in between `pass you` and `a soft voice`, and just after `says`, as per dialogue rules.